README.md

```diff
@@ -52,6 +52,7 @@ Application Options:
   -i, --instance= (required) Cloud Spanner Instance ID.
   -d, --database= (required) Cloud Spanner Database ID.
   -q, --quiet     Disable all interactive prompts.
+  -t, --target=   Table names to be deleted.
```

Comment:
How about using `--tables` for the long option name? It will be likely to have another option like `--ignore-tables` to exclude specific tables in the future, and I think it would be more natural if we use `--tables` for this option.